src/main/java/net/mcreator/element/types/LivingEntity.java

```diff
@@ -188,6 +189,19 @@ public LivingEntity(ModElement element) {
 
 		this.inventorySize = 9;
 		this.inventoryStackSize = 64;
+
+		this.modelLayers = new ArrayList<>();
+	}
+
+	public static class ModelLayerEntry {

```

Comment:
Does the model (layer JAVA model) usage detection here work correctly, given here model is just a string?

Reply:
You mean I forgot to add a method to retrieve the model stored here?

Reply:
I am not saying if you forgot anything, I am just asking if this will work because I have a feeling it won't.

How to fix this would be more suited to be decided by you given you have designed #3500 and how it should be used